### PR TITLE
SW-318.: add --port-mask and --baudrate option

### PR DESCRIPTION
**Pull request by fjmolinas**
This PR adds --port-mask and --baudrate option.

### CODE REVIEW

**Comment by fjmolinas** [comment]
@TimothyClaeys Rebased!

File: openvisualizer/main.py [hunk 1]
```diff
@@ -524,6 +526,15 @@ def _add_parser_args(parser):
         help='port mask for serial port detection, e.g \'/dev/tty/USB*\''
     )
 
+    parser.add_argument(
+        '--baudrate',
+        dest='baudrate',
+        type=list_from_string,
+        default='115200',
+        action='store',
+        help='List of baudrates to probe for, e.g \'115200 500000\''
```
Comment (TimothyClaeys) [inline openvisualizer/main.py]: if I tried to use the option as follows:
`openv-server --baudrate=115200 500000`
or 
`openv-server --baudrate 115200 500000`
it fails.

I think you should change this to:
`
parser.add_argument(
        '--baudrate',
        dest='baudrate',
        type=int,
        default=[115200],
        action='store',
        nargs='+',
        help='List of baudrates to probe for, e.g \'115200 500000\''
    )
`

File: openvisualizer/main.py [hunk 2]
```diff
@@ -566,6 +577,8 @@ def main():
 
     if args.port_mask:
         options.append('seial port mask         = {0}'.format(args.port_mask))
+    if args.baudrate:
+        options.append('Baudrates to probe      = {0}'.format(args.baudrate))
```
Comment (TimothyClaeys) [inline openvisualizer/main.py]: can you change this to a small letter, just for consistency 
Baudrates --> baudrates

File: openvisualizer/main.py [hunk 3]
```diff
@@ -566,6 +577,8 @@ def main():
 
     if args.port_mask:
         options.append('seial port mask         = {0}'.format(args.port_mask))
```
Comment (TimothyClaeys) [inline openvisualizer/main.py]: fix typo: seial --> serial

File: openvisualizer/motehandler/moteprobe/moteprobe.py [hunk 4]
```diff
@@ -85,18 +84,25 @@ def find_serial_ports(is_iot_motes=False, port_mask=None):
...
-            tester.set_timeout(2)
-            tester.test(blocking=True)
-            if tester.get_stats()['numOk'] >= 1:
-                mote_ports.append((port[0], BAUDRATE_LOCAL_BOARD))
-            probe.close()
-            probe.join()
-
+            try:
+                for baud in baudrate:
+                    log.debug("Probing port {} at baud {}".format(port, baud))
```
Comment (TimothyClaeys) [inline openvisualizer/motehandler/moteprobe/moteprobe.py]: fix: change "baud" to "baudrate"

File: openvisualizer/motehandler/moteprobe/moteprobe.py [hunk 5]
```diff
@@ -34,12 +34,11 @@
 
 # ============================ defines =========================================
 
-BAUDRATE_LOCAL_BOARD = 115200
 BAUDRATE_IOTLAB = 500000
 
 # ============================ functions =======================================
 
-def find_serial_ports(is_iot_motes=False, port_mask=None):
+def find_serial_ports(is_iot_motes=False, port_mask=None, baudrate=[115200]):
```
Comment (TimothyClaeys) [inline openvisualizer/motehandler/moteprobe/moteprobe.py]: the argument parser makes sure the variable "baudrate" already has the value [115200], so default value here is unnecessary?
Change to:
def find_serial_ports(baudrate, is_iot_motes=False, port_mask=None):

File: openvisualizer/main.py [hunk 6]
```diff
@@ -43,6 +43,27 @@
 coloredlogs.install(level='WARNING', logger=log, fmt='%(asctime)s [%(name)s:%(levelname)s] %(message)s',
                     datefmt='%H:%M:%S')
 
+# ============================ functions =======================================
+
```
Comment (TimothyClaeys) [inline openvisualizer/main.py]: this function is not necessary.
see https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse

File: openvisualizer/main.py [hunk 7]
```diff
@@ -493,6 +516,13 @@ def _add_parser_args(parser):
         help='port number'
     )
 
+    parser.add_argument(
```
Comment (TimothyClaeys) [inline openvisualizer/main.py:500]: Don't use "list_from_string", something like this should work I think:    

parser.add_argument(
        '--port-mask',
        dest='port_mask',
        type=str,
        action='store',
        nargs='+',
        help='List of baudrates to probe for, e.g \'115200 500000\''
    )

File: openvisualizer/motehandler/moteprobe/moteprobe.py [hunk 8]
```diff
@@ -50,33 +49,39 @@ def find_serial_ports(is_iot_motes=False):
...
-            for i in range(winreg.QueryInfoKey(key)[1]):
-                try:
-                    val = winreg.EnumValue(key, i)
-                except:
-                    pass
-                else:
-                    serial_ports.append((str(val[1]), BAUDRATE_LOCAL_BOARD))
-        except Exception:
-            pass
+    if port_mask == None:
```
Comment (TimothyClaeys) [inline openvisualizer/motehandler/moteprobe/moteprobe.py]: if port_mask is None

**Review by TimothyClaeys — CHANGES_REQUESTED** [review]
Left some remarks, let me know if something is unclear

**Comment by fjmolinas** [comment]
@TimothyClaeys I think I addressed your change requests, let me know if there is anything remaining!

**Comment by fjmolinas** [comment]
In my last push I believe I fixed the issues you mentioned offline. Testing procedure:

- `openv-server --baudrate 9600 115200 500000`

```
09:56:07 [OpenVisualizerServer:VERBOSE] Loading logging configuration: config/logging.conf
09:56:07 [OpenVisualizerServer:INFO] Initializing OV Server with options:
                - use page zero           = False
                - use VCD logger          = False
                - baudrates to probe      = ['9600', '115200', '500000']
09:56:09 [MoteProbe:WARNING] MoteProbe@/dev/ttyUSB1: invalid serial frame: (100B) 7e-9e-c2-ff-ff-ff-ff-ff-ff-1c-1e-9e-ff-ff-ff-ff-ff-ff-ff-ff-ff-c0-c0-e0-e0-3c-3c-1c-1e-ff-ff-3d-ff-ff-ff-ff-ff-ff-ff-e0-fc-ff-ff-ff-ff-ff-ff-fc-fc-f8-ff-ff-ff-ff-ff-ff-ff-3d-ff-ff-fc-1e-c0-ff-ff-ff-ff-ff-ff-ff-ff-0f-c0-c0-f8-ff-ff-ff-ff-ff-c0-c0-e0-e0-c0-c0-e0-e0-ff-ff-3d-ff-ff-ff-ff-ff-ff-e0-fc-7e wrong CRC
09:56:16 [MoteProbe:SUCCESS] discovered following serial-port(s): ['/dev/ttyUSB1@115200']
09:56:16 [OpenVisualizerServer:INFO] Extracting firmware definitions.
09:56:16 [Utils:VERBOSE] Extracting firmware component names
09:56:16 [Utils:VERBOSE] Generating firmware log descriptions.
09:56:16 [Utils:VERBOSE] Extracting 6top return codes.
09:56:16 [Utils:VERBOSE] Extracting 6top states.
09:56:16 [OpenVisualizerServer:INFO] Starting RPC server
09:56:16 [RPL:INFO] registering DAGroot 00-12-4b-00-14-b5-b5-af
```

- `openv-server --baudrate 9600 500000 115200`

```
09:57:39 [OpenVisualizerServer:VERBOSE] Loading logging configuration: config/logging.conf
09:57:39 [OpenVisualizerServer:INFO] Initializing OV Server with options:
                - use page zero           = False
                - use VCD logger          = False
                - baudrates to probe      = ['9600', '500000', '115200']
09:57:43 [MoteProbe:WARNING] MoteProbe@/dev/ttyUSB1: invalid serial frame: (2157B) 7e-ff-ff-ff-3d-fc-ff-ff-ff-ff-fc-9e-1e-ff-ff-ff-ff-ff-ff-ff-ff-ff-fe-cf-ff-ff-ff-ff-ff-3c-fa-f6-ff-e0-e0-e0-ff-ff-ff-ff-ff-ff-ff-ff-fc-e0-f0-f0-f0-ff-ff-ff-ff-ff-ff-ff-ff-e0-e0-f0-f0-ff-ff-ff-ff-e0-ff-ff-ff-fc-9e-1e-ff-ff-ff-ff-ff-ff-ff-ff-ff-ff-fe-cf-ff-ff-ff-ff-ff-ff-ff-3d-fa-f6-c0-c0-c0-fe-ff-ff-ff-ff-ff-ff-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-7e wrong CRC
09:57:49 [MoteProbe:SUCCESS] discovered following serial-port(s): ['/dev/ttyUSB1@115200']
09:57:49 [OpenVisualizerServer:INFO] Extracting firmware definitions.
09:57:49 [Utils:VERBOSE] Extracting firmware component names
09:57:49 [Utils:VERBOSE] Generating firmware log descriptions.
09:57:49 [Utils:VERBOSE] Extracting 6top return codes.
09:57:49 [Utils:VERBOSE] Extracting 6top states.
09:57:49 [OpenVisualizerServer:INFO] Starting RPC server
09:57:50 [RPL:INFO] registering DAGroot 00-12-4b-00-14-b5-b5-af
```

- `openv-server --baudrate 9600 500000`

```
09:59:45 [OpenVisualizerServer:VERBOSE] Loading logging configuration: config/logging.conf
09:59:45 [OpenVisualizerServer:INFO] Initializing OV Server with options:
                - use page zero           = False
                - use VCD logger          = False
                - baudrates to probe      = ['9600', '500000']
09:59:53 [MoteProbe:SUCCESS] discovered following serial-port(s): []
09:59:53 [OpenVisualizerServer:INFO] Extracting firmware definitions.
09:59:53 [Utils:VERBOSE] Extracting firmware component names
09:59:53 [Utils:VERBOSE] Generating firmware log descriptions.
09:59:53 [Utils:VERBOSE] Extracting 6top return codes.
09:59:53 [Utils:VERBOSE] Extracting 6top states.
09:59:53 [OpenVisualizerServer:INFO] Starting RPC server
```

- `openv-server --baudrate 115200`

```
10:00:15 [OpenVisualizerServer:VERBOSE] Loading logging configuration: config/logging.conf
10:00:15 [OpenVisualizerServer:INFO] Initializing OV Server with options:
                - use page zero           = False
                - use VCD logger          = False
                - baudrates to probe      = ['115200']
10:00:17 [MoteProbe:SUCCESS] discovered following serial-port(s): ['/dev/ttyUSB1@115200']
10:00:17 [OpenVisualizerServer:INFO] Extracting firmware definitions.
10:00:17 [Utils:VERBOSE] Extracting firmware component names
10:00:17 [Utils:VERBOSE] Generating firmware log descriptions.
10:00:17 [Utils:VERBOSE] Extracting 6top return codes.
10:00:17 [Utils:VERBOSE] Extracting 6top states.
10:00:17 [OpenVisualizerServer:INFO] Starting RPC server
10:00:18 [RPL:INFO] registering DAGroot 00-12-4b-00-14-b5-b5-af

```

- `openserver`

```
10:00:59 [OpenVisualizerServer:VERBOSE] Loading logging configuration: config/logging.conf
10:00:59 [OpenVisualizerServer:INFO] Initializing OV Server with options:
                - use page zero           = False
                - use VCD logger          = False
                - baudrates to probe      = [115200]
10:01:01 [MoteProbe:SUCCESS] discovered following serial-port(s): ['/dev/ttyUSB1@115200']
10:01:01 [OpenVisualizerServer:INFO] Extracting firmware definitions.
10:01:01 [Utils:VERBOSE] Extracting firmware component names
10:01:01 [Utils:VERBOSE] Generating firmware log descriptions.
10:01:01 [Utils:VERBOSE] Extracting 6top return codes.
10:01:01 [Utils:VERBOSE] Extracting 6top states.
10:01:01 [OpenVisualizerServer:INFO] Starting RPC server
10:01:02 [RPL:INFO] registering DAGroot 00-12-4b-00-14-b5-b5-af
```

- `openv-server --baudrate 9600 115200 500000 --port-mask /dev/riot/tty-openmote-b_*`

```
10:01:43 [OpenVisualizerServer:VERBOSE] Loading logging configuration: config/logging.conf
10:01:43 [OpenVisualizerServer:INFO] Initializing OV Server with options:
                - use page zero           = False
                - use VCD logger          = False
                - serial port mask        = ['/dev/riot/tty-openmote-b_0']
                - baudrates to probe      = ['9600', '115200', '500000']
10:01:45 [MoteProbe:WARNING] MoteProbe@/dev/riot/tty-openmote-b_0: invalid serial frame: (99B) 7e-ff-e0-ff-ff-ff-ff-ff-ff-f4-be-1e-fe-ff-ff-ff-ff-ff-ff-ff-ff-fe-cf-ff-ff-ff-3d-ff-ff-ff-ff-f2-e4-07-c0-c0-fc-ff-ff-ff-ff-ff-ff-ff-fc-c0-c0-e0-e0-fc-ff-ff-ff-ff-ff-ff-ff-ff-ff-3d-3c-3c-1e-9e-ff-d0-ff-ff-ff-ff-ff-ff-ff-ff-f4-be-1e-fe-ff-ff-ff-ff-ff-ff-ff-fe-cf-ff-ff-ff-ff-ff-3d-ff-ff-ff-f2-e6-7e wrong CRC
10:01:45 [MoteProbe:SUCCESS] discovered following serial-port(s): ['/dev/riot/tty-openmote-b_0@115200']
10:01:45 [OpenVisualizerServer:INFO] Extracting firmware definitions.
10:01:45 [Utils:VERBOSE] Extracting firmware component names
10:01:45 [Utils:VERBOSE] Generating firmware log descriptions.
10:01:45 [Utils:VERBOSE] Extracting 6top return codes.
10:01:45 [Utils:VERBOSE] Extracting 6top states.
10:01:45 [OpenVisualizerServer:INFO] Starting RPC server
10:01:46 [RPL:INFO] registering DAGroot 00-12-4b-00-14-b5-b5-af
```

- `openv-server --port-mask /dev/riot/tty-openmote-b_*`

```
10:02:23 [OpenVisualizerServer:VERBOSE] Loading logging configuration: config/logging.conf
10:02:23 [OpenVisualizerServer:INFO] Initializing OV Server with options:
                - use page zero           = False
                - use VCD logger          = False
                - serial port mask        = ['/dev/riot/tty-openmote-b_0']
                - baudrates to probe      = [115200]
10:02:23 [MoteProbe:SUCCESS] discovered following serial-port(s): ['/dev/riot/tty-openmote-b_0@115200']
10:02:23 [OpenVisualizerServer:INFO] Extracting firmware definitions.
10:02:23 [Utils:VERBOSE] Extracting firmware component names
10:02:23 [Utils:VERBOSE] Generating firmware log descriptions.
10:02:23 [Utils:VERBOSE] Extracting 6top return codes.
10:02:23 [Utils:VERBOSE] Extracting 6top states.
10:02:23 [OpenVisualizerServer:INFO] Starting RPC server
10:02:24 [RPL:INFO] registering DAGroot 00-12-4b-00-14-b5-b5-af
```

**Comment by fjmolinas** [comment]
Thanks for the review!

**Pull request merged**